### PR TITLE
Fix #659

### LINKED ISSUE #659: Add a Lyrics Finder
**Is your feature request related to a problem? Please describe.**
Most of us like to know the lyrics of the song we are listening to. So, this is a 'lyrics finder' app that will help you get lyrics of your favorite songs. Hence, I want to add this feature.

**Describe the solution you'd like**
This feature will be used by searching the song name. The user can search for the song name and the results will be shown and then the user can get the lyrics of the song they were searching for. This feature will be using an API.

**Additional context**
The following are the screenshots:
Lyrics Finder - Google Chrome 20-06-2021 19_20_10

Please assign me this issue under LGMSOC21

### PR DESCRIPTION
**Pull request by priyaastha**
## Explanation

This PR fixes issue #659 . This issue was regarding adding a feature. Through this pull request, I have added a Lyrics finder that uses musixmatch API. You can search for a song or an artist and it will show you the results accordingly.
  
Fixes #659 

## Checklist

- [x] The PR title starts with "Fix #issue_number".  
- [x] My branch is up-to-date with the main branch.
- [x] The PR does not contain any unnecessary auto-generated code.
- [x] The PR does not break any part of the existing code.
- [x] The PR creator have carefully read & adher to the guidelines of the project.
- [x] The PR is raised after the issue has been assigned to the PR creator.

### CODE REVIEW

**Review by GowthamGoush — CHANGES_REQUESTED** [review]
Hey, @priyaastha, wonderful work, but before we could merge, consider adding a readme as per #145.
Also, you may remove the changes that you've made in the "Project Website" folder.

**Comment by priyaastha** [comment]
@GowthamGoush sir, I made all the changes. Please review and merge. Thank you.

File: Project Website/assets/js/main.js [hunk 1]
```diff
@@ -938,4 +938,4 @@ function randomImage() {
   $('.myimage').append("<img src='" + path + imgs[i] + "'>").hide().fadeIn(2000);
 }
 
-randomImage();
```
Comment (GowthamGoush) [inline Project Website/assets/js/main.js:941]: This line of code has met some changes. Please consider reverting to the original.

**Review by GowthamGoush — CHANGES_REQUESTED** [review]
I've mentioned the line that interferes.

File: Project Website/assets/js/main.js (hunk 1 above)
Comment (priyaastha) [reply Project Website/assets/js/main.js:941]: The same line has been removed and added as shown in the screenshot.
If still, you think that I should revert it to the original, please guide me, sir.

Fix #659 by priyaastha · Pull Request #718 · Tejas1510_Hacking-Scripts - Google Chrome 25-06-2021 21_33_35

**Comment by GowthamGoush** [comment]
Alright, no worries, @priyaastha. 

The version control is susceptible to even minor changes. So, the newline at the end of the code might be a reason. Just try pasting the exact piece of code and hope this should work since I, too, don't find much of a change from the original file.

Also, please feel free to call by my name. I'm just a student😅

**Comment by GowthamGoush** [comment]
Any progress, @priyaastha?

**Comment by priyaastha** [comment]
@GowthamGoush  I have done the changes you asked for. Please check if it's okay now. Thank you.

**Comment by GowthamGoush** [comment]
Alright, the issue didn't resolve and seemed to be concerned with configuration. Anyway, I'll merge this time.

**Review by GowthamGoush — APPROVED** [review]
Ready to be merged.

**Pull request merged**